lib/modules/manager/bundler/artifacts.spec.ts

```diff
@@ -37,6 +37,7 @@ const adminConfig: RepoGlobalConfig = {
   localDir: join('/tmp/github/some/repo'),
   cacheDir: join('/tmp/cache'),
   containerbaseDir: join('/tmp/cache/containerbase'),
+  dockerSidecarImage: 'ghcr.io/containerbase/sidecar:8.0.2',
```

Comment:
```suggestion
  dockerSidecarImage: 'ghcr.io/containerbase/sidecar',
```
i would prefer this to reduce test changes